pom.xml

```diff
@@ -39,6 +39,8 @@
         <org.testng.version>6.1.1</org.testng.version>
         <jets3t.version>0.9.4</jets3t.version>
         <bouncycastle.version>1.55</bouncycastle.version>
+        <integration.base.version>1.0.2</integration.base.version>
+        <skip-tests>true</skip-tests>
```

Comment:
Why are we skipping the tests here?

Reply:
For the build in jenkins usually, we skip test cases